### DISCUSSION TITLE
TTS of numbers with german model.

### DISCUSSION THREAD

**Discussion by cstarzinger** [espnet/espnet#4993] [General]
Hello,

I trained a German TTS model which works fine so far. 
But when I want to create speech from a string of numbers e.g.: "1, 2 , 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20" it is not working correctly. 
The string is converted to the English number words "one, two, three, four, five, ..." before the audio is generated with German pronunciation.

[123.zip](https://github.com/espnet/espnet/files/10943500/123.zip)

It would be great if someone could point me in the right direction to change this behavior.

Thanks!
C.

**Comment by kan-bayashi** [comment]
Maybe you used `cleaner=tacotron`? It will convert expand numbers.
`cleaner=none` may solve the issue.
